docs/user/bots.rst

```diff
@@ -2559,15 +2559,22 @@ specified with quotes. Following operators may be used to match events:
 If part of a rule matches the given conditions, the actions enclosed in `{` and
 `}` are applied. By default, all events that are matched or not matched by rules
 in the sieve file will be forwarded to the next bot in the pipeline, unless the
-`drop` action is applied.
+`drop` action is applied. 
+
+**ATTENTION** Math only supported on DateTime objects currently!
+In addtion `add`, `add!` and `update` do also support super basic math operations `+` and `-`.
 
  * `add` adds a key value pair to the event. This action only applies if the key is not yet defined in the event. If the key is already defined, the action is ignored. Example:
 
    ```add comment = 'hello, world'```
 
+   Basic math operations
+   ```add time.observation += '1 hour'```
+   ```add time.observation -= '10 hours'```
+
  * `add!` same as above, but will force overwrite the key in the event.
 
- * `update` modifies an existing value for a key. Only applies if the key is already defined. If the key is not defined in the event, this action is ignored.  Example:
+ * `update` modifies an existing value for a key. Only applies if the key is already defined. If the key is not defined in the event, this action is ignored. Supports mathematical expressions like above. Example:
```

Comment:
Can you add an example here as well please?

Reply:
*This* only applies...
*This* supports...